lab-07-chris/server.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const bodyParser = require('./lib/body-parser');
+const http = require('http');
+const url = require('url');
+const queryString = require('querystring');
+const cowsay = require('cowsay');
+const PORT = process.env.PORT || 3000;
+
+const server = module.exports = http.createServer(function(req, res) {
+  req.url = url.parse(req.url);
+  req.url.query = queryString.parse(req.url.query);
+
+  if(req.method === 'POST') {
+
+    if(req.url.pathname === '/cowsay') {
```

Comment:
This applies to Lines 16 and 32. You're not properly getting your 400 error with the way this is written. Modify your if statements to the following:

```
if(req.url.pathname === '/cowsay' && req.body) 
```